pipeline.py

```diff
@@ -23,12 +22,15 @@
 GENERATED_STANDALONE_FILE_NAME = "standalone.py"
 DEFAULT_REPO_URL = "https://github.com/instructlab/taxonomy.git"
 KFP_MODEL_SERVER_CM = "sdg/kfp-model-server.yaml"
-BASE_MODE = "ibm-granite/granite-7b-base"
+BASE_MODEL = "ibm-granite/granite-7b-base"
 BASE_MODEL_DIR = "/data/model/"  # <- "model ID for vLLM chat/completions - corresponds to path within pvc"
+
+# eval args
 MMLU_TASKS_LIST = "mmlu_anatomy,mmlu_astronomy"
 MODEL_DTYPE = "bfloat16"
 FEW_SHOTS = 5
-BATCH_SIZE = 8
+# BATCH_SIZE can also be an int, for example "8" is converted to an int in eval/final
+BATCH_SIZE = "auto"
```

Comment:
do you want to make this change ins the standalone.tmp as well? https://github.com/redhat-et/ilab-on-ocp/blob/e5a53233ce1d7f0a010fea3eb9595c9861cfa881/standalone/standalone.tpl#L100

Reply:
oh shoot, yup - should I change to "auto" or "8" wdyt?

Reply:
I updated standalone.tpl (and standalone.py) to `BATCH_SIZE = "auto"`